git-conventional-commits.default.yaml

```diff
@@ -15,6 +15,10 @@ convention:
   - revert
   commitScopes: []
   releaseTagGlobPattern: v[0-9]*.[0-9]*.[0-9]*
+  msgRegex: /^(?<type>\w+)(?:\((?<scope>[^()]+)\))?(?<breaking>!)?:\s*(?<description>.+)/i
```

Comment:
I would not add those configs to the default config because it's an edge case. I would rather just add a section to the readme file for that edge case. Just because it makes default setup a little cumbersome.